### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by aaroncolichia**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] All tests passed: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript: 30 points

**Checkpoint 2**
- [ ] The application works as it should: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript and the DOM API: 30 points

**Checkpoint 3**
- [ ] Use of React: 25 points
- [ ] Accesses an API: 25 points
- [ ] Proper use of documentation (commenting on code): 25 points
- [ ] The application functions as it should: 25 points

### CODE REVIEW

File: 02week/pigLatin.js [hunk 1]
```diff
@@ -7,14 +7,50 @@ const rl = readline.createInterface({
...
-  // Your code here
 
+// have to use function def to get access to arguments var
+function pigLatin() {
+  const vowels = ['a', 'e', 'i', 'o', 'u'];
+  let response = '';
+  for (i=0; i < arguments.length; i++) {
+    if (typeof arguments[i] !== 'string') {
+      console.log('argument ' + arguments[i] + ' is not a string');
+      return false;
```
Comment (reneedudley) [inline 02week/pigLatin.js]: re-write this so you don't return false, you don't want to ever evalute if something is true and return false, it doesn't look great and it's hard to follow

File: 02week/pigLatin.js [hunk 2]
```diff
@@ -7,14 +7,50 @@ const rl = readline.createInterface({
...
+    }
+    let segment = arguments[i].toLowerCase();
+    let first = '';
```
Comment (reneedudley) [inline 02week/pigLatin.js]: It's unnecessary to define these as lets outside of the else, it wastes space and opens up scope. You can define them as const on line 40-42

File: 02week/pigLatin.js (hunk 1 above)
Comment (aaroncolichia) [reply 02week/pigLatin.js]: Also switched to "for of" es6 style

File: 02week/pigLatin.js (hunk 2 above)
Comment (aaroncolichia) [reply 02week/pigLatin.js]: thanks
